manifests/t/Trufflesuite/Ganache/Beta/2.6.0-beta.3/Trufflesuite.Ganache.Beta.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Trufflesuite.Ganache.Beta
+PackageVersion: 2.6.0-beta.3
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/trufflesuite/ganache/releases/download/v2.6.0-beta.3/Ganache-2.6.0-beta.3-win-setup.exe
+  InstallerSha256: 5CD27D240AAC2AEB216AA2CE6AA5CA11D04D7DFAED5E160C2FA16C898969C46F
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: appx
+  InstallerUrl: https://github.com/trufflesuite/ganache/releases/download/v2.6.0-beta.3/Ganache-2.6.0-beta.3-win-x64.appx
+  InstallerSha256: E3C679BF6A116A6C6C873CCCED39D15E0F1CF21CC507F5039097DA5B0898FC96
+  SignatureSha256: 6F4A37E93C09413936AA850854747A081189F339C749A968D627D51BF488EC40
+  Scope: machine
```

```suggestion
  Scope: user
```
Created another user, it's actually `Scope: user`.